config/locales/de.yml

```diff
@@ -136,6 +146,18 @@ de:
       away: "Team 2"
       score: "Ergebnis"
       points: "Punkte"
+    metric:
+      elo: "ELO"
+      win_loss: "Gewinne-Niederlagen"
+      true_skill: "Wahres Können"
+    gamemode:
+      ko: "KO"
+      ko_group: "Gruppen-KO"
+      double_elimination: "Doppel-KO-System"
+      round_robin: "Runder Robin"
```

Comment:
According to Wikipedia:

> Ein Runder Robin, englisch Round-robin, ist eine Bitt- und Beschwerdeschrift, auch Petition genannt, die von ihren Verfassern und Befürwortern rundherum in einem Kreis unterschrieben worden ist

I don't think that's what was meant. Other translation include "Jeder-gegen-Jeden" or "Rundenturnier". Even just keeping the english original "Round Robin" would have been more explanatory.
#19 states "Jeder-gegen-Jeden" to be the wanted translation.